_maintenance_update_graph_data.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+outFile="$(dirname $0)/js/graphdata.tsv"
+
+curl --silent https://ct.tacticalsecret.com/graphdata.tsv >${outFile}
```

Comment:
Could you name graphdata.tsv certcounts.tsv or similar? That way if we add additional graphs using this method, we can distinguish them.